### DISCUSSION TITLE
Project 4 part 3

### DISCUSSION THREAD

**Discussion by taa18009** [cooperrc/computational-mechanics#71] [Q&A]
Does anyone have any suggestions or resources for getting started on [problem 3 in project 4](https://cooperrc.github.io/computational-mechanics/projects/04_Linear_Algebra-project.html#determine-cross-sectional-area)? I'm not really sure where to start, and I don't see any example problems similar to this problem. I may be missing something though.

**Comment by cooperrc** [comment]
In Project_04, you solve a finite element problem to find the displacements and forces in a structure. 

Problem 3 asks you to find the cross-sectional area so that deflection is <0.2 mm

There are 2 approaches:
1. use the [root-finding](https://cooperrc.github.io/computational-mechanics/module_03/04_Getting_to_the_root.html) methods by defining a function that returns maximum deflection - 0.2 given a cross-sectional area e.g. `deflection(A)` then use a bracketing method or modified secant method
2. because the problem is _linear_, $EA\cdot \mathbf{Kx= F}$, you can use the existing solution to find the ideal solution e.g. if when `A = 0.1` max deflection was 1 mm, then I would multiply A by 5, `A = 0.5` to make the structure $\times 5$ stiffer and the max deflection should decrease to 0.2 mm
